### PR TITLE
[Naitei][Ecommerce]   View Wishlist

### PR DESCRIPTION
**Pull request by SuKay712**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/78393

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
Screenshot from 2024-08-04 15-11-17
Screenshot from 2024-08-04 15-11-27

### CODE REVIEW

**Comment by SuKay712** [comment]
ready

**Comment by SuKay712** [comment]
ready

File: app/views/users/edit.html.erb [hunk 1]
```diff
@@ -10,35 +10,7 @@
...
-          <%= link_to "#" do %>
-            <i class="list__icon fa-solid fa-right-from-bracket"></i>
-            <span><%= t "profile.logout" %></span>
-          <% end %>
-        </div>
-      </div>
-    </div>
+    <%= render "layouts/sidebar" %>
     <div class="formedit">
       <label for="" class="formedit__title"><%= t("profile.personal_info") %></label>
```
Comment (ThaiKhacDuoc) [inline app/views/users/edit.html.erb]: Bỏ ngoặc ()
```suggestion
      <label for="" class="formedit__title"><%= t "profile.personal_info" %></label>
```

**Review by ThaiKhacDuoc — COMMENTED** [review]
cmt

File: app/views/wishlists/_wishlist.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (ghost) [inline app/views/wishlists/_wishlist.html.erb]: bỏ dòng trống

**Review by ghost — COMMENTED** [review]
cmt nhỏ fix rồi merged

File: app/views/wishlists/_wishlist.html.erb (hunk 2 above)
Comment (SuKay712) [reply app/views/wishlists/_wishlist.html.erb]: dạ rồi ạ

**Comment by SuKay712** [comment]
ready

**Comment by SuKay712** [comment]
ready

**Pull request merged**